Makefile

```diff
@@ -114,18 +113,21 @@ build: gen-semver fmt vet ## Build manager binary.
 run: generate gen-semver fmt vet static-manifests ## Run a controller from your host.
 	go run ./main.go
 
-podman-build: gen-semver build-base-image ## Build podman image with the manager.
-	podman build . -t ${DEFAULT_IMG} --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
+podman-build: gen-semver download-csm-common ## Build podman image with the manager.
+	$(eval include csm-common.mk)
+	podman build --pull . -t ${DEFAULT_IMG} --build-arg BASEIMAGE=$(CSM_BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
 
-podman-build-no-cache: gen-semver build-base-image ## Build podman image with the manager.
-	podman build --no-cache . -t ${DEFAULT_IMG} --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
+podman-build-no-cache: gen-semver download-csm-common ## Build podman image with the manager.
+	$(eval include csm-common.mk)
+	podman build --pull --no-cache . -t ${DEFAULT_IMG} --build-arg BASEIMAGE=$(CSM_BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
 
 podman-push: podman-build ## Builds, tags and pushes docker image with the manager.
 	podman tag ${DEFAULT_IMG} ${IMG}
 	podman push ${IMG}
 
-docker-build: gen-semver build-base-image ## Build docker image with the manager.
-	docker build . -t ${DEFAULT_IMG} --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
+docker-build: gen-semver download-csm-common ## Build docker image with the manager.
+	$(eval include csm-common.mk)
+	docker build --pull . -t ${DEFAULT_IMG} --build-arg BASEIMAGE=$(CSM_BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE)
```

Comment:
Does this require docker? podman is used elsewhere.

Reply:
This Makefile has targets that use podman and docker. A few repos do that. I asked everyone if docker is required. A future improvement is to standardize on podman for all builds.